packages/p2p/src/pages/my-ads/ads-table/my-ads-row.tsx

```diff
@@ -0,0 +1,333 @@
+import React from 'react';
+import classNames from 'classnames';
+import {
+    DesktopWrapper,
+    HorizontalSwipe,
+    Icon,
+    MobileWrapper,
+    Popover,
+    ProgressIndicator,
+    Table,
+    Text,
+} from '@deriv/components';
+import { isMobile, formatMoney } from '@deriv/shared';
+import { observer } from '@deriv/stores';
+import { Localize, localize } from 'Components/i18next';
+import { buy_sell } from 'Constants/buy-sell';
+import { ad_type } from 'Constants/floating-rate';
+import AdStatus from 'Pages/my-ads/ad-status';
+import { useStores } from 'Stores';
+import { TAdDetails } from 'Types';
+import { generateEffectiveRate } from 'Utils/format-value';
+import AdType from '../ad-type';
+
+const MyAdsRow = observer(({ row: advert }: TAdDetails) => {
+    const { floating_rate_store, general_store, my_ads_store, my_profile_store } = useStores();
```

Comment:
```suggestion
    const { floating_rate_store: { exchange_rate, ..... }, general_store, my_ads_store, my_profile_store } = useStores();
```

It's better to destructure these store's variables!